src/Modules/SharedModule/components/NavBar/NavBar.tsx

```diff
@@ -60,18 +58,28 @@ export default function NavBar() {
                     >
                       <div className="user-data d-flex  align-items-center ">
                         <div className="user-img rounded-circle me-3">
-                          <img
-                            className="w-100 rounded-circle me-1"
-                            src={Images.test}
-                            alt=""
-                          />
+                          {currentUser?.imagePath ? (
+                            <div className={`${style.navUserImg} text-white `}>
+                              <img
+                                className="w-100 "
+                                src={`https://upskilling-egypt.com:3003/${currentUser.imagePath}`}
+                                alt=""
```

Comment:
We can make the alt text the username and adding avatar after it => "Omar's avatar"